apps/deploy_monitoring/docker-compose.deploy.latest.yml

```diff
@@ -1,6 +1,18 @@
 version: "3"
 
 services:
+  tezedge-memprof:
```

Comment:
@vlad9486 @adonagy @jurajselep 
do we want to run memprof also on master.dev.tezedge.com?
if so, we need to add this also to `docker-compose.deploy.latest-release.yml`  right?

Reply:
@bkontur You are right we will run it on master. I'll add.